code/datums/diseases/advance/symptoms/rodman.dm

```diff
@@ -0,0 +1,123 @@
+	if(A.resistance >= 10)
+		severity -= 2 //Becomes a healing symptom at this point.
+	if((A.stealth >= 2) && (A.stage_rate >= 12))
+		severity += 3 //Ghost role creation turns this into a more risky symptom, but not directly deadly. Likely that the resistance threshold is hit too, so it should end up as a 2 severity total.
+
+/datum/symptom/rodman_syndrome/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.resistance >= 10)
+		restore_mind = TRUE
+	if(A.transmission >= 8)
+		increase_volume = TRUE
+	if((A.stealth >= 2) && (A.stage_rate >= 12))
+		awaken_monkey = TRUE
+	if(!ishuman(A.affected_mob)) //Important to know if they are a monkey from the very start.
+		is_monkey = TRUE
+			
+		
+/datum/symptom/rodman_syndrome/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+
+	switch(A.stage)
+		if(2, 3)
+			if(prob(20))
+				to_chat(M, "<span class='notice'>[pick(
+				"You feel a need to return to the past.",
+				"You wonder if the ceilings are high enough for trees.",
+				"A desire to screech comes over you.", 
+				"Bananas sound good right about now.",
+				"Ook?",
+				"Perhaps those apes were right...",
+				"Your thoughts seem to be clearing up.")]</span>")
+		if(4, 5)
+			if(is_monkey == FALSE)
+				if(restore_mind == TRUE)
+					M.adjustOrganLoss(ORGAN_SLOT_BRAIN, -3) //Only heals brain damage at a power equal to Mind Restoration but does nothing for traumas.
+					if(prob(5))
+						to_chat(M, "<span class='notice'>[pick(
+						"Your thoughts seem to be clearing up.",
+						"Your brain feels great.",
+						"The fog in your mind is clearing up.")]</span>")
+					return
+				if((increase_volume == TRUE) && (prob(35)))
+					M.say( pick( list(
+					"Ook!!", 
+					"Ook.", 
+					"Ooh-ooh!!",
+					"Ah-ah!!",
+					"Eek-eek!!",
+					"Hoo hoo!!", 
+					"SCREE!!") )) //More volume means shouted messages now
+					playsound(M.loc, pick( list('sound/creatures/monkey/monkey_screech_1.ogg','sound/creatures/monkey/monkey_screech_2.ogg','sound/creatures/monkey/monkey_screech_3.ogg','sound/creatures/monkey/monkey_screech_4.ogg','sound/creatures/monkey/monkey_screech_5.ogg','sound/creatures/monkey/monkey_screech_6.ogg','sound/creatures/monkey/monkey_screech_7.ogg')), 100, 1)
+					return
+				if(prob(20))
+					M.say( pick( list(
+					"Ook!",
+					"Ook.",
+					"Eek", 
+					"Oop?", 
+					"Aak-eek.", 
+					"Chee."
+					) )) //Return to monke.
+					playsound(M.loc, pick( list('sound/creatures/monkey/monkey_screech_1.ogg','sound/creatures/monkey/monkey_screech_2.ogg','sound/creatures/monkey/monkey_screech_3.ogg','sound/creatures/monkey/monkey_screech_4.ogg','sound/creatures/monkey/monkey_screech_5.ogg','sound/creatures/monkey/monkey_screech_6.ogg','sound/creatures/monkey/monkey_screech_7.ogg')), 50, 1)
+			if((!ishuman(M)) && (awaken_monkey == TRUE) && (is_monkey == TRUE)) //Confirm they were a monkey from the start, currently a monkey and that the threshold is hit.

```

Comment:
You dont need to check if they are equal to true and false, just having the variable there will check to see if its equal to anything that isnt null or zero (in this case FALSE is zero and TRUE is one) so, you can just have it like this:
```suggestion
			if(!ishuman(M) && awaken_monkey && is_monkey) //Confirm they were a monkey from the start, currently a monkey and that the threshold is hit.
```
And if you wanted to check if it was false (null or zero), just put a ! before it. Also, you dont need to encase everything in parentheses, unless you are grouping multiple things together for the purposes of && and || (you still have to encase it in the ones for the if statement).

Reply:
Gotcha.
Was encapsulating everything for the sake of my own readability, so I knew they were completely separate sections.